en/components/excel_library_working_with_sparklines.md

```diff
@@ -1,32 +1,38 @@
----
+﻿---
 title: Excel Library Component - Native Angular | Ignite UI for Angular
-_description: The Ignite UI for Excel Library component ... TODO.
+_description: The Ignite UI for Excel Library component with Sparkline support.
 _keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Excel Library, Angular Excel Library Example, Angular Excel Library Component, Angular Excel Engine
 ---
 ## Excel Library Working with Sparklines
 
-TODO
+This topic provides conceptual information about Sparkline support available on the `ignite-angular-excel`
 
```

Comment:
remove this sentence, add more content that is useful for the cystomer. this is a template senstnce that isn't providing any value to the topic